libs/vm/include/vm/variant.hpp

```diff
@@ -475,37 +475,23 @@ struct Variant
   }
 };
 
-struct TemplateParameter1 : public Variant
+template <std::size_t ID = 0>
```

Comment:
Why default to 0? (as in, why have a default at all?)

Reply:
The very same question can beasked about all default values across the whole project.
Daefault is default, and zero makes the most sense based all aspects in given context.
I don't mind really, can be dropped.